core/trino-main/src/main/java/io/trino/sql/planner/LocalExecutionPlanner.java

```diff
@@ -514,6 +514,7 @@ public LocalExecutionPlan plan(
             TypeProvider types,
             PartitioningScheme partitioningScheme,
             List<PlanNodeId> partitionedSourceOrder,
+            Optional<Integer> hashPartitionCount,
```

Comment:
I don't see corresponding change in `io.trino.execution.SqlTaskExecutionFactory#create`